scripts/build-debs.sh

```diff
@@ -43,8 +43,36 @@ export CONTAINER="fpf.local/sd-client-builder-${DEBIAN_VERSION}"
 
 . ./scripts/image_prep.sh
 
+# We're going to store artifacts in a temp directory
+BUILD_DEST=$(mktemp -d)
+
 $OCI_BIN run --rm $OCI_RUN_ARGUMENTS \
     -v "${BUILDER}:/builder:Z" \
+    -v "${BUILD_DEST}:/build:Z" \
     --env NIGHTLY="${NIGHTLY:-}" \
     --entrypoint "/src/scripts/build-debs-real.sh" \
     $CONTAINER
+
+ls "$BUILD_DEST"
+# Copy the build artifacts to our project's /build
+mkdir -p build
+cp ${BUILD_DEST}/* build/
+
+FAST="${FAST:-}"
+if [[ -z $FAST ]]; then
+    CONTAINER2="fpf.local/sd-client-lintian"
+    $OCI_BIN build scripts/lintian -t $CONTAINER2
+    # Display verbose info, and fail on warnings and errors.
+    # We suppress mismatched-override because of sublte differences in
```

Comment:
Typo:

```suggestion
    # We suppress mismatched-override because of subtle differences in
```

Reply:
in fact, this comment is outdated because previously I had it try to exclude the `mismatched-override` tag but removed it later, so dropped this whole comment